cisco_secure_email_threat_defense/README.md

```diff
@@ -37,15 +37,15 @@ Configure the Datadog endpoint to forward Cisco Secure Email Threat Defense logs
 | API Key           | API Key from Cisco Secure Email Threat Defense.         |
 | Client ID      | Client ID from Cisco Secure Email Threat Defense.    |
 | Client Password      | Client Password from Cisco Secure Email Threat Defense.    |
-| Verdict Delay      | Time (in minutes) required to allow verdicts to be updated. Specify a number from 0 to 720. The Default value is 30.    |
+| Verdict Delay      | Time (in minutes) required to allow verdicts to be updated. Specify a number from 0 to 720. The default value is 30.    |
 
 ## Data Collected
 
 ### Logs
 
 The Cisco Secure Email Threat Defense integration collects and forwards Cisco Secure Email Threat Defense message logs to Datadog. This integration will ingest messages with verdict values of scam, malicious, phishing, BEC, spam, graymail, and neutral.
 
-**Note**: Events are fetched with a delay according to the time specified in the Verdict Delay. This delay is intended to accommodate retrospective verdicts. However, this does not guarantee that all retrospective verdicts are processed within this timeframe, as the time required for updates varies.
+**Note**: Events are fetched every xx minutes up to the time specified in the Verdict Delay. This delay is necessary to ensure that the logs include retrospective verdicts. However, this does not guarantee that all retrospective verdicts are captured within this timeframe, as the time required for updates can vary. For complete verdict information, please log in to your Cisco Secure Email Threat Defense system.
```

Comment:
To make sure I understand, is it the case that events are fetched at least as often as the verdict delay, but might be fetched more often? I'm confused about `xx minutes up to` and I suspect it needs to be reworded.

Reply:
@neko-dd 
I think a better statement to explain this would be
"Events are fetched every xx minutes, incorporating a delay defined by the user known as the Verdict Delay."
So lets suppose the crawler runs every 5 minutes, and verdict delay is of 2 minutes.
If I am running my integration at 12:00, I will get data up till 11:58.
Again, the integration will fetch data at 12:05 , and will give data up till 12:03., and so on.

I hope this explains the scenario. Please let me know in case of any doubt.

Reply:
Is XX not defined in a way that we can reveal to the user?

Reply:
@jnhunsberger 
Can you please help here?
We feel that the original statement seems perfect to define our case "Events are fetched with a delay according to the time specified in the Verdict Delay."
Please let us know your thoughts here.

Reply:
@ankitarajput-crest Is the verdict delay user configurable? If so, then your copy can work. If it is static and only changeable with a code update, we should state the number of minutes in the tile copy. Either way we should not ship this with "xx" in the copy of the tile.

Reply:
Since verdict delay parameter is user configurable we have kept first statement as `Events are fetched with a delay according to the time specified in the Verdict Delay.`